avidemux_de.ts

```diff
@@ -381,37 +362,37 @@ Drop timing informations ?</source>
     <message>
         <location filename="../ADM_userInterfaces/ADM_dialog/processing.ui" line="+14"/>
         <source>Dialog</source>
-        <translation type="unfinished">Dialog</translation>
+        <translation>Dialog</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>TimeLeft</source>
-        <translation type="unfinished">Restzeit</translation>
+        <translation>Verbleibende Zeit</translation>
     </message>
     <message>
         <location line="+64"/>
         <source>0</source>
-        <translation type="unfinished">0</translation>
+        <translation>0</translation>
     </message>
     <message>
         <location line="-51"/>
         <source>00:00:00</source>
-        <translation type="unfinished">00:00:00</translation>
+        <translation>00:00:00</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>ProcessImages</source>
-        <translation type="unfinished"></translation>
+        <translation>Verarbeitete Bilder</translation>
     </message>
     <message>
         <location line="-7"/>
         <source>Speed</source>
-        <translation type="unfinished"></translation>
+        <translation>Geschwindigkeit</translation>
     </message>
     <message>
         <location line="-14"/>
         <source>0 fps</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
I am not sure that it is safe to rely on empty string as translation to be always replaced by the source string. Especially marking such translations as finished makes me feel uncomfortable. I may be easily wrong here, a reference to the relevant place in the documentation is welcome.

Reply:
I think it should be safe:
https://doc.qt.io/qt-5/linguist-translators.html#selecting-string-to-translate
_No Translation: The string does not have a translation. Click the icon to accept the empty translation anyway_
The docs also says that if the source will be changed, it should be reset to "Not Accepted"

But of course, I can change it to “real” translations, if you feel safer with that :)